.github/workflows/linkchecker.yaml

```diff
@@ -0,0 +1,26 @@
+name: Check links
+
+on:
+  schedule:
+    - cron: "10 7 * * *"
```

Comment:
Maybe in the same way that we did for the webbot, this could run during weekdays? ` cron: "10 7 * * 1-5"`

Reply:
You think? Unlike with your pull request notifier, I think we want to know when tests fail any day of the week.

Reply:
hopefully yeah. I am fine in running it every day.